[Pulsar Manager]Add release notes for puslar manager 0.2.0 (#8085)
apache/pulsar: site2/website/pulsar-manager-release-notes.md

## Apache Pulsar Manager

### 0.2.0 &mdash; 2020-09-28 <a id="0.2.0"></a>

> #### Fixes

Reviewer: I suggest that we just remove this heading.
For guides, refer to [heading usage](https://developers.google.com/style/headings)

* Support sub and unsub operations [PR-240](https://github.com/apache/pulsar-manager/pull/240).
* Support peeking messages from the Pulsar broker [PR-241](https://github.com/apache/pulsar-manager/pull/241).
* Support BookKeeper visual manager 1.2.0 [PR-300](https://github.com/apache/pulsar-manager/pull/300).
* Support forwarding messages through HTTPS [PR-258](https://github.com/apache/pulsar-manager/pull/258).
* Support displaying stats for tenants and namespaces [PR-243](https://github.com/apache/pulsar-manager/pull/243).
* Add a configuration file for the backend service of Pulsar manager [PR-236](https://github.com/apache/pulsar-manager/pull/236).
* Add default configurations for the environment [PR-242](https://github.com/apache/pulsar-manager/pull/242).
* Fixe an SQL syntax error [PR-298](https://github.com/apache/pulsar-manager/pull/298).
* Fix the issue that Pulsar Manager fail to process the request sent to the Pulsar proxy [PR-281](https://github.com/apache/pulsar-manager/pull/281).
* Change the default port and replace the request URI [PR-316](https://github.com/apache/pulsar-manager/pull/316).

### 0.1.0 &mdash; 2019-11-25 <a id="0.1.0"></a>

> #### Fixes

Reviewer: remove the heading.

## Apache Pulsar Manager

### 0.2.0 &mdash; 2020-09-28 <a id="0.2.0"></a>

#### Fixes

* Support multi address for broker stats [#306](https://github.com/apache/pulsar-manager/pull/306)
* Use PulsarAdmin instead of HttpUtil [#315](https://github.com/apache/pulsar-manager/pull/315)
* Serve frontend directly from Pulsar Manager backend process [#288](https://github.com/apache/pulsar-manager/pull/288)

> * Support docker for jwt [#218](https://github.com/apache/pulsar-manager/pull/218)

Reviewer 2: ```suggestion
* Support docker for JWT [#218](https://github.com/apache/pulsar-manager/pull/218)
```

> * Support sub and unsub [#240](https://github.com/apache/pulsar-manager/pull/240)

Reviewer 2: ```suggestion
* Support sub and unsub operations [PR-240](https://github.com/apache/pulsar-manager/pull/240).
```

> * Support peek messages from Ppulsar broker [#241](https://github.com/apache/pulsar-manager/pull/241)

Reviewer 2: ```suggestion
* Support peeking messages from the Pulsar broker [PR-241](https://github.com/apache/pulsar-manager/pull/241).
```

> * Support BbookKeeper visual manager 1.2.0 [#300](https://github.com/apache/pulsar-manager/pull/300)

Reviewer 2: ```suggestion
* Support BookKeeper visual manager 1.2.0 [PR-300](https://github.com/apache/pulsar-manager/pull/300).
```

> * Support https forward [#258](https://github.com/apache/pulsar-manager/pull/258)

Reviewer 2: Support forwarding messages through HTTPS?

> * Support display stats for tenant and namespace [#243](https://github.com/apache/pulsar-manager/pull/243)

Reviewer 2: ```suggestion
* Support displaying stats for tenants and namespaces [PR-243](https://github.com/apache/pulsar-manager/pull/243).
```

> * Add a configuration file for backend service of pulsar manager [#236](https://github.com/apache/pulsar-manager/pull/236)

Reviewer 2: ```suggestion
* Add a configuration file for the backend service of Pulsar manager [PR-236](https://github.com/apache/pulsar-manager/pull/236).
```

> * Add default configuration for environment [#242](https://github.com/apache/pulsar-manager/pull/242)

Reviewer 2: ```suggestion
* Add default configurations for the environment [PR-242](https://github.com/apache/pulsar-manager/pull/242).
```

> * fixed a SQL syntax error [#298](https://github.com/apache/pulsar-manager/pull/298)

Reviewer 2: ```suggestion
* Fixe an SQL syntax error [PR-298](https://github.com/apache/pulsar-manager/pull/298).
```

> * Fix the pulsar-manager can not process the request by pulsar proxy [#281](https://github.com/apache/pulsar-manager/pull/281)

Reviewer 2: ```suggestion
* Fix the issue that Pulsar Manager fail to process the request sent to the Pulsar proxy [PR-281](https://github.com/apache/pulsar-manager/pull/281).
```
